### ISSUE TITLE
Loading of ressources failed

### ISSUE THREAD

**Issue by Portugao** [zikula-modules/MediaModule#99]
```
Laden von gemischten aktiven Inhalten "http://cdn-files.deezer.com/js/min/dz.js" wurde blockiert.[Weitere Informationen] erstellen
Laden von gemischten aktiven Inhalten "http://cdn-files.deezer.com/js/min/dz.js" wurde blockiert.[Weitere Informationen] erstellen
Laden fehlgeschlagen für das <script> mit der Quelle "http://cdn-files.deezer.com/js/min/dz.js".
```
Auf https Seite....

**Comment by Guite** [comment]
In theory this would be easy to fix in `Resources/views/MeiaType/Deezer/webCreation.html.twig`. But the problem is that the URL https://cdn-files.deezer.com/js/min/dz.js does not work.

**Comment by Portugao** [comment]
Puh. That is not so good.

**Comment by Guite** [comment]
Fixed in 51f4f7b2e4de20ef41b95e446c90927d63a7a8cf
